src/main/java/fr/insee/survey/datacollectionmanagement/query/controller/QuestioningController.java

```diff
@@ -98,9 +97,6 @@ public ResponseEntity<?> getQuestioningsBySurveyUnit(@PathVariable("id") String
 
     }
 
-    @PreAuthorize("@AuthorizeMethodDecider.isInternalUser() "
```

Comment:
no preauthorize in new version ?

Reply:
@PreAuthorize(AuthorityPrivileges.HAS_MANAGEMENT_PRIVILEGES) already on top of controller